src/models/extension.ts

```diff
@@ -1,7 +1,7 @@
 import type { BaseModel } from "./base";
 
-export interface ExtensionInterface extends BaseModel {
+export interface ExtensionInterface<T = any> extends BaseModel {
```

Comment:
Would `T` be ever any other type rather than `any` ?

Reply:
Extension can include any value, primitive, object, array.

Reply:
From slack:

You can have any value in extension, we cannot say what exactly type is in extension, but user can cast extension value to given type, so he/she can do this:
```
const extension = model.extensions().get<TwitterExtensionsShape>('twitter');
```

and then:
```
extensions.value()
```
will return

```
TwitterExtensionsShape
```

type.

Reply:
Yeah. Makes totally sense. Thanks for clarifying!